backend/oasst_backend/prompt_repository.py

```diff
@@ -800,6 +800,15 @@ def fetch_message_text_labels(self, message_id: UUID, user_id: Optional[UUID] =
             query = query.filter(TextLabels.user_id == user_id)
         return query.all()
 
+    def fetch_message_revision_history(self, message_id: UUID) -> list[MessageRevision]:
+        revisions: list[MessageRevision] = sorted(
+            self.db.query(MessageRevision).filter(MessageRevision.message_id == message_id).all(),
+            key=lambda revision: revision.id.int >> 80,
```

Comment:
Could you maybe add a comment here that we sort by the date part of the uuid7? Might not be directly obvious to everyone...